src/commands/dev/dev.js

```diff
@@ -486,8 +511,10 @@ const createDevCommand = (program) => {
       'netlify dev',
       'netlify dev -d public',
       'netlify dev -c "hugo server -w" --targetPort 1313',
+      'netlify dev --graph',
```

Comment:
drive-by adding a new example

Reply:
You'll need to run `npm run doc` to sync the docs with this change (and fix the CI)